target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -11,6 +11,11 @@ board=$(board_name)
 boardname="${board##*,}"
 
 case "$board" in
+buffalo,wxr-2533dhp)
+	ucidef_set_led_usbport "usb" "USB" "${boardname}:green:usb" "usb1-port1" "usb2-port1" "usb3-port1" "usb4-port1"
```

Comment:
You could try to see if you can declare the usbport trigger via DT. 
https://github.com/openwrt/openwrt/blob/0d23fd2ab29a66f5d03187db4fac3e396b4f3b62/target/linux/ath79/dts/qca9558_tplink_archer-c7.dtsi#L41